lang/nl.yml

```diff
@@ -5,54 +5,181 @@ nl:
     ADDEMAILRECIPIENT: 'Email ontvanger toevoegen'
     CLEARBUTTON: Wissen
     CONFIGURATION: Configuratie
+    DISABLEAUTHENICATEDFINISHACTION: 'Bedanktpagina toegankelijk maken zonder formulier-inzending'
     DISABLECSRFSECURITYTOKEN: 'CSRF-token uitschakelen'
     DISPLAYERRORMESSAGESATTOP: 'Toon foutmeldingen bovenaan het formulier'
     ENABLELIVEVALIDATION: 'Zet real-time validatie aan'
     NORECIPIENTS: 'Let op: Er zijn geen ontvangers ingesteld, mogelijk mist u hiermee inzendingen.'
     ONCOMPLETELABEL: 'Toon wanneer gereed'
-    RECIPIENTS: Ontvangers
+    RECIPIENTS: E-mail
     SAVESUBMISSIONS: 'Formulier-inzendingen niet opslaan in het CMS'
-    SHOWCLEARFORM: 'Toon knop om formulier te legen'
+    SHOWCLEARFORM: 'Toon herstel-knop om formulier te legen'
     SUBMISSIONS: Inzendingen
     SUBMITBUTTON: Verzenden
-    TEXTONCLEAR: 'Tekst op herstel button:'
+    TEXTONCLEAR: 'Tekst op herstel-knop'
     TEXTONSUBMIT: 'Tekst op Verzend-button'
   SilverStripe\UserForms\Control\UserDefinedFormAdmin:
     CANCEL: Annuleren
     CONFIRM_FOLDER_LABEL_A: 'Bestanden die via een formulier ingestuurd worden, moeten zorgvuldig worden opgeslagen om de niet publiek toegankelijk te maken. Zorg ervoor dat de map die u selecteert alleen kan worden bekeken door de juiste gebruikers. Map-rechten kunnen worden beheerd in het onderdeel Bestanden.'
+    CONFIRM_FOLDER_LABEL_B: 'The folder selected will become the default for this form. Het kan op een individueel <i>upload-veld</i> worden aangepast.'
```

Comment:
That translation is partially in English.

Reply:
Sharp! Just updated it in Transifex (and one other typo (in CHECKEDBYDEFAULT))